packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Delete.tsx

```diff
@@ -1,7 +1,18 @@
 import React, { MouseEvent } from 'react';
+import VERSION from '@codesandbox/common/lib/version';
 import { useOvermind } from 'app/overmind';
 import { css } from '@styled-system/css';
-import { Button, Stack } from '@codesandbox/components';
+import { Button, Stack, Text, Element, Link } from '@codesandbox/components';
+import { SpectrumLogo, GithubIcon, TwitterIcon } from './icons';
+
+const links = [
+  { href: 'https://twitter.com/codesandbox', icon: <TwitterIcon /> },
```

Comment:
```suggestion
  { href: 'https://twitter.com/codesandbox', Icon: TwitterIcon },
```

Reply:
There were some issues with that I can't remeber what but calling it as a function fixed them